### PR TITLE
Flower.

### PR DESCRIPTION
**Pull request by BlackShadow1**
@ZombiePower

### CODE REVIEW

File: src/pocketmine/block/Flower.php [hunk 1]
```diff
@@ -93,4 +93,7 @@ public function onUpdate($type){
 
 		return false;
 	}
+        public function getDrops(Item $item){
+            return [$this->id, $this->meta, 1];
```
Comment (thebigsmileXD) [inline src/pocketmine/block/Flower.php]: $this->id is not defined.. i think

**Comment by BlackShadow1** [comment]
I think is, look Slab.php

**Comment by ZombiePower** [comment]
No no.... Omfg.... -_- @BlackShadow1 because they have DIFFERENT IDS

**Comment by ZombiePower** [comment]
works?

**Comment by BlackShadow1** [comment]
Yes.

**Pull request merged**